.github/workflows/release.yml

```diff
@@ -29,11 +29,17 @@ jobs:
             mkdir libgit2/build && cd libgit2/build
             cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local
             sudo cmake --build . --target install
-        - name: Compile splitsh
+        - name: Clone splitsh/lite
+          uses: actions/checkout@master
+          with:
+            repository: splitsh/lite
+            ref: v2.0.0
```

Comment:
One question - are we sure we want `v2.0.0` here? Looks like it was released a year ago, so we're probably ok

https://github.com/splitsh/lite/releases

Reply:
Is not mandatory to have this version, but:
1.- Is the latest
2.- It works with lib2lib 1.5, which is the version we are currently using. 

I debater weather or not using `master` in here but I have a suspicious feeling that that's how we ended with this step failing in the first place? But of course I do not have proof of that.